coderd/userauth.go

```diff
@@ -761,6 +778,63 @@ func (api *API) userOIDC(rw http.ResponseWriter, r *http.Request) {
 	http.Redirect(rw, r, redirect, http.StatusTemporaryRedirect)
 }
 
+// claimFields returns the sorted list of fields in the claims map.
+func claimFields(claims map[string]interface{}) []string {
+	var fields []string
+	for field := range claims {
+		fields = append(fields, field)
+	}
+	sort.Strings(fields)
+	return fields
+}
+
+// blankFields returns the list of fields in the claims map that are
+// an empty string.
+func blankFields(claims map[string]interface{}) []string {
+	fields := make([]string, 0)
```

Comment:
Suggestion: Simplify

```suggestion
	var fields []string
```

Reply:
This ends up making its way to `slog.F`; I initially had what you suggested but this allocates a `nil` slice which ends up being JSON-slogged as `null` instead of `[]`, which I would expect to be confusing to the viewer. See here: https://go.dev/play/p/kE1TVgwAYUz

Reply:
Ah, makes sense. In that case I would still suggest `fields := []string{}` as a simpler form to `make`, giving a zero non-nil value.